### PR TITLE
Initial Commit of a MEAN stack web page

### PR DESCRIPTION
**Pull request by jjalvare**

### CODE REVIEW

File: terraform/mean-stack-ibmcloud/main.tf [hunk 1]
```diff
@@ -0,0 +1,52 @@
+module  "simple_instance" {
+ source = "github.com/jjalvare/simple_vm"
```
Comment (jwcroppe) [inline terraform/mean-stack-ibmcloud/main.tf]: Should we also push this to the DevOps folder as well?

File: terraform/mean-stack-ibmcloud/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Deploy a MEAN stack using Terraform on IBM Cloud VPC 
+
+Deploy a Mongodb Express Angular Nodejs (MEAN) stack on Ubuntu 18.04 ppc64le running in IBM Cloud VPC
```
Comment (jwcroppe) [inline terraform/mean-stack-ibmcloud/README.md]: s/Mongodb/MongoDB

File: terraform/mean-stack-ibmcloud/variables.tf [hunk 3]
```diff
@@ -0,0 +1,42 @@
+################################################################
```
Comment (jwcroppe) [inline terraform/mean-stack-ibmcloud/variables.tf:1]: Be consistent with copyright files in all tf files and shell scripts?

File: terraform/modules/simple_vm/versions.tf [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (jwcroppe) [inline terraform/modules/simple_vm/versions.tf:15]: Delete leading newline

File: terraform/mean-stack-ibmcloud/main.tf (hunk 1 above)
Comment (jjalvare) [reply terraform/mean-stack-ibmcloud/main.tf]: Is added in this PR. I will later make it point to this repo as terraform does not allow me to specify a directory tree other than master.

**Pull request merged**